### PR TITLE
Windows cleanup

### PR DESCRIPTION
**Pull request by hrydgard**
Trying various things that may help #7957, also removing some unused code.

### CODE REVIEW

File: GPU/GLES/Framebuffer.cpp [hunk 1]
```diff
@@ -1656,13 +1656,7 @@ void FramebufferManager::PackFramebufferSync_(VirtualFramebuffer *vfb, int x, in
 void FramebufferManager::EndFrame() {
 	if (resized_) {
 		DestroyAllFBOs();
-		glstate.viewport.set(0, 0, PSP_CoreParameter().pixelWidth, PSP_CoreParameter().pixelHeight);
-		int zoom = g_Config.iInternalResolution;
-		if (zoom == 0) // auto mode
-			zoom = (PSP_CoreParameter().pixelWidth + 479) / 480;
```
Comment (unknownbrackets) [inline GPU/GLES/Framebuffer.cpp]: Is this already handled by other code on all platforms?

-[Unknown]

File: Windows/MainWindowMenu.cpp [hunk 2]
```diff
@@ -328,30 +386,30 @@ namespace MainWindow {
 		}
 	}
 
-	void SaveStateActionFinished(bool result, void *userdata) {
+	static void SaveStateActionFinished(bool result, void *userdata) {
 		PostMessage(MainWindow::GetHWND(), WM_USER_SAVESTATE_FINISH, 0, 0);
 	}
 
-
+	// not static
```
Comment (unknownbrackets) [inline Windows/MainWindowMenu.cpp]: Why?  Doesn't really look different from `setTexScalingType`, which is static?

-[Unknown]

File: GPU/GLES/Framebuffer.cpp (hunk 1 above)
Comment (hrydgard) [inline GPU/GLES/Framebuffer.cpp]: Ugh, yeah, you are right, we need it here for them. But on Windows it's duplication. I'll ifdef it.

File: Windows/MainWindowMenu.cpp (hunk 2 above)
Comment (hrydgard) [inline Windows/MainWindowMenu.cpp]: As you saw I moved the menu code out into its own file, but the main file still wants to call this function. Didn't want to rock the boat, cleanup later.

**Pull request merged**